opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -124,48 +125,133 @@ message MetricDescriptor {
+  // value" for every data point. It can be used when an "unknown" aggregation.
+  // 
+  // It does not contain a temporality field (even if it may be known) because
+  // the aggregation is unknown so points cannot be combined using the same
+  // aggregation to aggregate over different time window. Because of this
+  // "StartTimeUnixNano" is ignored for all data points.
+  //
+  // A Metric of this Type MUST store its values as Int64DataPoint or
+  // DoubleDataPoint.
+  message Gauge {
+    // It describes the value type of the measurement used to build this
+    // aggregation.
     //
-    // A Metric of this Type MUST store its values as Int64DataPoint.
-    INT64 = 1;
+    // Determines if the points are Int64DataPoint or DoubleDataPoint, as well
+    // as the value type of the exemplars.
+    MeasurementValueType measurement_value_type = 1;
+  }
 
-    // MONOTONIC_INT64 values are monotonically increasing signed 64-bit
-    // integers.
+  // Represents the type of a numeric scalar metric that is calculated as a sum
+  // of all reported measurements over a time interval.
+  //
+  // TODO: Decide if this should support only MonotonicSum
+  // https://github.com/open-telemetry/opentelemetry-specification/issues/725.
+  //
+  // A Metric of this Type MUST store its values as Int64DataPoint or
+  // DoubleDataPoint.
+  message Sum {
+    // It describes the value type of the measurement used to build this
+    // aggregation.
     //
-    // A Metric of this Type MUST store its values as Int64DataPoint.
-    MONOTONIC_INT64 = 2;
+    // Determines if the points are Int64DataPoint or DoubleDataPoint, as well
+    // as the value type of the exemplars.
+    MeasurementValueType measurement_value_type = 1;
+
+    // Temporality is the temporal quality values of a metric have.
+    Temporality temporality = 2;
+
+    // If "true" means that the sum is monotonic.
+    bool is_monotonic = 3;
+
+    // TODO: Decide if knowing that the aggregated measurements are raw
+    // measurements is important or not. Equivalent of "Snapshot" in #162.
+  }
 
-    // DOUBLE values are double-precision floating-point numbers.
+  // Represents the type of a metric that is calculated by aggregating as a
+  // Histogram of all reported measurements over a time interval.
+  //
+  // A Metric of this Type MUST store its values as HistogramDataPoint.
+  message Histogram {
+    // It describes the value type of the measurement used to build this
+    // aggregation.
     //
-    // A Metric of this Type MUST store its values as DoubleDataPoint.
-    DOUBLE = 3;
+    // Determines the value type of the exemplars.
+    MeasurementValueType measurement_value_type = 1;
+
+    // Temporality is the temporal quality values of a metric have.
+    Temporality temporality = 2;
 
-    // MONOTONIC_DOUBLE values are monotonically increasing double-precision
-    // floating-point numbers.
+    // TODO: Decide if knowing that the aggregated measurements are raw
```

Comment:
> Decide if knowing that the aggregated measurements are raw measurements

Should this be "decide if knowing that the metric values are aggregated from raw measurements is important" (similar for below)

Reply:
Done.